Failed to `npm install`

`package.json` is invalid json. As a consequence, `npm install` fails with the following output:

```sh
npm info it worked if it ends with ok
npm info using npm@3.10.8
npm info using node@v6.9.1
npm ERR! Linux 3.16.0-4-amd64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! file /sources/package.json
npm ERR! code EJSONPARSE

npm ERR! Failed to parse json
npm ERR! Trailing comma in object at 21:3
npm ERR!   }
npm ERR!   ^
npm ERR! File: /sources/package.json
npm ERR! Failed to parse package.json data.
npm ERR! package.json must be actual JSON, not just JavaScript.
npm ERR! 
npm ERR! This is not a bug in npm.
npm ERR! Tell the package author to fix their package.json file. JSON.parse

npm ERR! Please include the following file with any support request:
npm ERR!     /sources/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -17,6 +17,6 @@
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.5.0",
     "babel-preset-react": "^6.5.0",
-    "webpack": "^1.12.14",
+    "webpack": "^1.12.14"
   }
 }

